pg_search/src/postgres/customscan/pdbscan/exec_methods/fast_fields/mixed.rs

```diff
@@ -1130,17 +836,14 @@ mod multi_field_collector {
             let ctid = self.ctid_ff.as_u64(doc).expect("ctid should be present");
             let scored = SearchIndexScore::new(ctid, score);
 
-            // Collect string fields - group by term ordinal for efficiency
-            for (field_idx, (_, str_column)) in self.string_columns.iter().enumerate() {
+            // Collect string fields
+            for (string_column_idx, (_, str_column)) in self.string_columns.iter().enumerate() {
                 let term_ord = str_column.term_ords(doc).next().unwrap_or(0);
```

Comment:
This converts a null to the empty string: there is a TODO in `StringFF` about this as well.

Reply:
Should we add the same TODO here, too (and also reference it)?

Reply:
Yea, will do. Would rather not change behavior during #2610 and #2612, but then will come back for it.